### PR TITLE
Relicensing: update .mailmap

### PR DESCRIPTION
**Pull request by tamiko**
Update .mailmap to fix a number of spelling mistakes and e-mail addresses

 - Mailmap: sort and compactify
 - Mailmap: add missing entries

 In reference to #16665

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -10,110 +10,88 @@
...
+Dakshina Ilangovan <dakshinai@tamu.edu> <dakshina.ilangova@0785d39b-7218-0410-832d-ea1e28bc413d>
+Lise-Marie Imbert-Gerard <lmig@math.umd.edu>
+Lise-Marie Imbert-Gerard <lmig@math.umd.edu> <imbertgerard@cims.nyu.edu>
+Vladimir Ivannikov <vladimir.ivannikov@hzg.de> <vovannikov@yandex.ru>
+Manu Jayadharan <maj136@pitt.edu> <35903705+mjayadharan@users.noreply.github.com>
+Daniel Jodlbauer <pmnotxor@gmx.at> <jdschule@gmx.at>
+Eldar Khattatov <eld.khattatov@gmail.com>
+Eldar Khattatov <elk58@pitt.edu>
+Sebastian Kinnewig <sebastian@kinnewig.org> <35812030+kinnewig@users.noreply.github.com>
+Sebastian Kinnewig <sebastian@kinnewig.org> <Sebastian@Kinnewig.org>
```
Comment (marcfehling) [inline .mailmap:58]:
```suggestion
```
new/correct mail is identical to old/incorrect mail

File: .mailmap [hunk 2]
```diff
@@ -10,110 +10,88 @@
...
+Sam Cox <spc29@le.ac.uk>
+Marc Fehling <marc.fehling@gmx.net>
+Rebecca Fildes <rfildes@ucdavis.edu>
+Menno Fraters <menno.fraters@hotmail.com>
+Menno Fraters <menno.fraters@outlook.com>
+Nicola Giuliani <ngiuliani@sissa.it> <nicolagiuliani@dhcp-129-199.sissa.it>
+Anne Glerum <a.c.glerum@uu.nl>
+Krishnakumar Gopalakrishnan <krishna.kumar@ucl.ac.uk> <krishnakumarg1984@users.noreply.github.com>
+Krishnakumar Gopalakrishnan <krishna.kumar@ucl.ac.uk> <krishnak@vt.edu>
 Alexander Grayver <agrayver@erdw.ethz.ch>
```
Comment (marcfehling) [inline .mailmap:42]:
```suggestion
```
duplicate (see line below)

File: .mailmap [hunk 3]
```diff
@@ -10,110 +10,88 @@
...
+Eldar Khattatov <elk58@pitt.edu>
+Sebastian Kinnewig <sebastian@kinnewig.org> <35812030+kinnewig@users.noreply.github.com>
+Sebastian Kinnewig <sebastian@kinnewig.org> <Sebastian@Kinnewig.org>
+Ingo Kligge <ingo.kligge@tu-dortmund.de>
+Alex Kokomov <kokomovalex@gmail.com>
+Adam Kosik <adam.kosik@atlas.cz>
+Konstantin Ladutenko <kostyfisik@gmail.com> <kostyfisik@users.noreply.github.com>
+Adam Lee <adam4130@users.noreply.github.com>
+Pablo Lucero <pablo.lucero@iwr.uni-heidelberg.de>
+unknown luzpaz <luzpaz@users.noreply.github.com>
```
[the remark this answers is not in the document]
Comment (tamiko) [reply .mailmap:65]: I wanted to drop this entry

File: .mailmap (hunk 1 above)
Comment (tamiko) [reply .mailmap:56]: I kept this one so that the e-mail is all lowercase.

**Comment by tamiko** [comment]
@marcfehling Updated except one case. Regarding users with incomplete name: we have a couple more, see spreadsheet. I suggest we make an effort to reach them to inquire about full names.

**Review by marcfehling — APPROVED** [review]

**Pull request merged**